tmt/steps/provision/local.py

```diff
@@ -136,7 +148,30 @@ def execute(
         )
 
     def install_scripts(self, scripts: Sequence[tmt.steps.scripts.Script]) -> None:
-        self.debug("No installation of tmt scripts is needed on localhost.")
+        """Install scripts required by tmt"""
+
+        # Make sure scripts directory exists
+        command = Command("mkdir", "-p", f"{self.scripts_path}")
+
+        if not self.facts.is_superuser:
+            command = Command("sudo") + command
+
+        self.execute(command, silent=True)
+
+        # Install all scripts on local guest
+        for script in scripts:
+            if not script.enabled(self):
+                continue
+
+            with script as source:
+                for filename in [script.source_filename, *script.aliases]:
+                    destination = script.destination_path or self.scripts_path / filename
+
+                    # Copy file and set permissions
+                    install_cmd = Command('install', '-p', '-m', '755', source, destination)
```

Comment:
there should be assertions somewhere for this

Reply:
Added a new test !